package-lock.json

```diff
@@ -1,6448 +1,8 @@
 {
   "name": "fx-private-relay-add-on",
-  "version": "2.4.3",
+  "version": "2.4.4",
```

Comment:
Oh, I thought I had to rebase because of this line, but looks like the version in the lockfile wasn't aligned with the one in `package.json` yet. Thanks dependabot for fixing that :)

@dependabot merge